packages/lesswrong/components/tagging/EditTagPage.tsx

```diff
@@ -14,7 +14,7 @@ export const EditTagForm = ({tag, successCallback, cancelCallback}: {
     collection={Tags}
     documentId={tag._id}
     queryFragment={getFragment('TagEditFragment')}
-    mutationFragment={getFragment('TagEditFragment')}
+    mutationFragment={getFragment('TagFragment')}
```

Comment:
can you explain a bit about what's going on here and why the query and mutation fragments are different?

Reply:
They are almost always different. The query fragment determines what data is used to populate the form. The mutation fragment determines what data comes back after you submit the form. The first one usually includes lots of data for the revision and editor fields, and the second one are just the resulting data fields.